src/entity/posts/PostAnalyticsHistory.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  OneToOne,
+  PrimaryColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+import type { Post } from './Post';
+
+@Entity()
+export class PostAnalyticsHistory {
+  @PrimaryColumn({ type: 'text' })
+  id: string;
+
+  @CreateDateColumn()
+  createdAt: Date;
+
+  @UpdateDateColumn()
+  updatedAt: Date;
+
+  @Column()
+  date: string;
+
+  @Column({ default: 0 })
+  impressions: number;
```

Comment:
Also storing only impressions for now, adding other columns will be possible since we store raw events and can always compute historical data if we want to